### PR TITLE
[IMP] new design : left menu reorganization

### PR DESCRIPTION
**Pull request by tivisse**
TASK : https://www.odoo.com/web#id=12927&view_type=form&model=project.task&action=333&active_id=248
PAD : https://pad.odoo.com/p/r.b68kIBheop1ptvzQ

### CODE REVIEW

File: addons/crm/security/crm_security.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
         <field name="groups_id" eval="[(4,ref('base.group_sale_manager')),(4,ref('base.group_sale_salesman'))]"/>
     </record>
 
-    <record model="ir.ui.menu" id="base.menu_base_config">
```
Comment (tivisse) [inline addons/crm/security/crm_security.xml]: Maybe a bad idea to change the xmlid ?

File: addons/hr_attendance/hr_attendance_view.xml [hunk 2]
```diff
@@ -107,16 +107,19 @@
             <field name="view_id"></field>  <!-- force empty -->
         </record>
 
-        <menuitem id="menu_hr_time_tracking" name="Time Tracking" parent="hr.menu_hr_root" sequence="5" groups="base.group_user,base.group_hr_user,base.group_hr_manager"/>
+        <menuitem id="timesheet_menu_root" name="Timesheets" sequence="10" icon="fa-user" groups="base.group_user,base.group_hr_user,base.group_hr_manager"/>
+        <menuitem id="menu_hr_time_tracking" name="Time Tracking" parent="timesheet_menu_root" sequence="5" />
 
-        <menuitem id="menu_hr_attendance" name="Attendances" parent="hr.menu_hr_root" sequence="10" groups="base.group_hr_attendance"/>
```
Comment (tivisse) [inline addons/hr_attendance/hr_attendance_view.xml]: Maybe a bad idea to change the xmlid ?

File: addons/hr_evaluation/hr_evaluation_view.xml [hunk 3]
```diff
@@ -71,9 +71,10 @@
             <field name="context">{'search_default_user_to_review_id': [active_id], 'default_user_to_review_id': active_id}</field>
         </record>
 
-        <menuitem name="Appraisal" parent="hr.menu_hr_root" id="menu_eval_hr" sequence="25"/>
```
Comment (tivisse) [inline addons/hr_evaluation/hr_evaluation_view.xml]: Maybe a bad idea to change the xmlid ?

File: addons/hr_expense/report/hr_expense_report_view.xml [hunk 4]
```diff
@@ -67,7 +67,7 @@
         <field name="search_view_id" ref="view_hr_expense_report_search"/>
     </record>
 
-    <menuitem action="action_hr_expense_report_all" id="menu_hr_expense_report_all" parent="hr.menu_hr_reporting" sequence="1"/>
```
Comment (tivisse) [inline addons/hr_expense/report/hr_expense_report_view.xml]: Maybe a bad idea to change the xmlid ?

File: addons/hr_payroll/hr_payroll_view.xml [hunk 5]
```diff
@@ -3,9 +3,9 @@
     <data>
         <!-- Root Menus -->
 
-        <menuitem id="payroll_configure" parent="hr.menu_hr_configuration" name="Payroll" groups="base.group_hr_manager" sequence="45"/>
-        <menuitem id="menu_hr_root_payroll" parent="hr.menu_hr_root" name="Payroll" sequence="30"/>
```
Comment (tivisse) [inline addons/hr_payroll/hr_payroll_view.xml]: Maybe a bad idea to change the xmlid ?

File: addons/hr_timesheet_sheet/hr_timesheet_sheet_view.xml [hunk 6]
```diff
@@ -214,8 +214,9 @@
             </field>
         </record>
 
-        <menuitem action="act_hr_timesheet_sheet_form" id="menu_act_hr_timesheet_sheet_form" parent="hr_attendance.menu_hr_time_tracking"
```
Comment (tivisse) [inline addons/hr_timesheet_sheet/hr_timesheet_sheet_view.xml]: Maybe a bad idea to change the xmlid ?

File: addons/marketing_campaign/marketing_campaign_view.xml [hunk 7]
```diff
@@ -184,9 +184,11 @@
         </field>
     </record>
 
-    <menuitem name="Lead Automation" id="menu_marketing_campaign" parent="base.marketing_menu" groups="marketing.group_marketing_user,marketing.group_marketing_manager"
```
Comment (tivisse) [inline addons/marketing_campaign/marketing_campaign_view.xml]: Maybe a bad idea to change the xmlid ?

File: addons/mass_mailing/views/mass_mailing.xml [hunk 8]
```diff
@@ -645,11 +646,11 @@
             <field name="view_mode">tree,form</field>
         </record>
 
-        <menuitem name="Mass Mailing" id= "menu_view_mass_mailing_config"
```
Comment (tivisse) [inline addons/mass_mailing/views/mass_mailing.xml]: Maybe a bad idea to change the xmlid ?

File: addons/point_of_sale/point_of_sale_view.xml [hunk 9]
```diff
@@ -989,8 +987,8 @@
         </record>
 
         <menuitem action="action_pos_session_opening"
-            parent="menu_point_of_sale"
-            id="menu_pos_session_opening" sequence="0"/>
```
Comment (tivisse) [inline addons/point_of_sale/point_of_sale_view.xml]: Maybe a bad idea to change the xmlid ?

File: addons/product_margin/product_margin_view.xml [hunk 10]
```diff
@@ -86,8 +86,8 @@
             </field>
         </record>
 
-  <menuitem id="base.next_id_73" name="Purchase" parent="base.menu_reporting" sequence="36"/>
```
Comment (tivisse) [inline addons/product_margin/product_margin_view.xml]: Maybe a bad idea to change the xmlid ?

File: addons/project/res_config_view.xml [hunk 11]
```diff
@@ -102,8 +102,8 @@
             <field name="target">inline</field>
         </record>
 
-        <menuitem id="base.menu_project_config" name="Project" parent="base.menu_config"
```
Comment (tivisse) [inline addons/project/res_config_view.xml]: Maybe a bad idea to change the xmlid ?

File: addons/purchase/report/purchase_report_view.xml [hunk 12]
```diff
@@ -94,21 +94,9 @@
         <field name="help">Purchase Analysis allows you to easily check and analyse your company purchase history and performance. From this menu you can track your negotiation performance, the delivery performance of your suppliers, etc.</field>
     </record>
 
-    <menuitem id="base.next_id_73" name="Purchase" parent="base.menu_reporting" sequence="37"
```
Comment (tivisse) [inline addons/purchase/report/purchase_report_view.xml]: Maybe a bad idea to change the xmlid ?

File: addons/purchase/res_config_view.xml [hunk 13]
```diff
@@ -104,8 +104,8 @@
             <field name="target">inline</field>
         </record>
 
-        <menuitem id="menu_purchase_config" name="Purchases" parent="base.menu_config"
```
Comment (tivisse) [inline addons/purchase/res_config_view.xml]: Maybe a bad idea to change the xmlid ?

File: addons/sales_team/sales_team.xml [hunk 14]
```diff
@@ -191,7 +191,7 @@
             </xpath>
         </template>
 
-        <menuitem id="base.next_id_64" name="Sales" parent="base.menu_reporting" sequence="1" groups="base.group_sale_manager"/>
```
Comment (tivisse) [inline addons/sales_team/sales_team.xml]: Maybe a bad idea to change the xmlid ?

File: addons/stock/stock_view.xml [hunk 15]
```diff
@@ -1774,13 +1776,13 @@
...
+        <menuitem id="menu_stock_sched" name="Schedulers" parent="stock.menu_stock_root" sequence="50" groups="stock.group_stock_manager"/>
         <menuitem action="action_procurement_compute" id="menu_procurement_compute" parent="menu_stock_sched" groups="base.group_no_one" sequence="2"/>
         <menuitem action="procurement.action_compute_schedulers" id="menu_stock_proc_schedulers" parent="menu_stock_sched" sequence="1" groups="stock.group_stock_manager"/>
         <menuitem action="procurement.procurement_exceptions" id="menu_stock_procurement_action" parent="menu_stock_sched" sequence="50" groups="stock.group_stock_manager"/>
-        <menuitem id="menu_stock_procurement" name="Automatic Procurements" parent="stock.menu_stock_configuration" sequence="5"/>
+        <menuitem id="menu_stock_procurement" name="Automatic Procurements" parent="stock.menu_stock_config_settings" sequence="5"/>
         <menuitem action="action_orderpoint_form" id="menu_stock_order_points" parent="stock.menu_reordering_rules_config" sequence="1"/>
-        <menuitem id="stock.next_id_61" name="Warehouse" sequence="40" parent="base.menu_reporting" groups="group_stock_manager"/>
```
Comment (tivisse) [inline addons/stock/stock_view.xml]: Maybe a bad idea to change the xmlid ?

**Comment by rim-odoo** [comment]
@tivisse  Yes, avoid changing xmlids, as it will add migration work

**Comment by rim-odoo** [comment]
@tivisse Can you make a more complete commit message listing the main modules/files split/moves ?

**Comment by tivisse** [comment]
@rim-odoo, @tde-banana-odoo seems ok to change/update/give_a_super_cute xmlid :)
Generally, I created new xmlid when I splitted some modules into several apps. 
I think that I have changed some ridiculous xmlid as next_id_64 or something like this. 

By the way, I have updated the commit message, I hope it is good enough ;)

File: addons/website_slides/__openerp__.py [hunk 16]
```diff
@@ -31,4 +31,6 @@
         'data/website_slides_demo.xml'
     ],
     'installable': True,
+    'auto_install': False,
```
Comment (rim-odoo) [inline addons/website_slides/__openerp__.py]: default value...

File: addons/website_links/static/src/js/website_links_backend.js [hunk 17]
```diff
@@ -0,0 +1,21 @@
+
+odoo.define('web.website_links_embedded', function (require) {
+"use strict";
+var core = require('web.core');
+var Widget = require('web.IFrameWidget');
```
Comment (rim-odoo) [inline addons/website_links/static/src/js/website_links_backend.js]: Perfect example of bad design: add an iFrame of the frontend inside the backend...

File: addons/web/views/webclient_templates.xml [hunk 18]
```diff
@@ -196,8 +196,9 @@
                 <t t-foreach="menu_data['children']" t-as="menu">
                     <div style="display: none" class="oe_secondary_menu" t-att-data-menu-parent="menu['id']">
                         <t t-foreach="menu['children']" t-as="menu">
-                            <div class="oe_secondary_menu_section">
-                                <t t-esc="menu['name']"/>
+                            <div class="oe_secondary_menu_section">                                
+                                <t t-if="menu['children']"><t t-esc="menu['name']"/></t>
+                                <t t-if="not menu['children']"><t t-call="web.menu_link"/></t>
```
Comment (rim-odoo) [inline addons/web/views/webclient_templates.xml]: Clickable secondary menus are ugly... need a restyling?

File: addons/project_issue/project_issue.py [hunk 19]
```diff
@@ -34,17 +34,6 @@
...
-    _name = "project.issue.version"
-    _order = "name desc"
-    _columns = {
-        'name': fields.char('Version Number', required=True),
-        'active': fields.boolean('Active', required=False),
-    }
-    _defaults = {
-        'active': 1,
-    }
-
```
Comment (rim-odoo) [inline addons/project_issue/project_issue.py]: :+1:

File: addons/mass_mailing/views/res_partner.xml [hunk 20]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0"?>
-<openerp>
-    <data>
-
-    </data>
-</openerp>
```
Comment (rim-odoo) [inline addons/mass_mailing/views/res_partner.xml]: :+1:

File: addons/contacts/__openerp__.py [hunk 21]
```diff
@@ -0,0 +1,44 @@
...
+    ],
+    'data': [
+        'contacts_view.xml',
+    ],
+    'installable': True,
+    'application': True,
+    'auto_install': False,
+}
+
+# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Comment (rim-odoo) [inline addons/contacts/__openerp__.py]: modelines

File: addons/contacts/__openerp__.py [hunk 22]
```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2004-2010 Tiny SPRL (<http://tiny.be>).
```
Comment (rim-odoo) [inline addons/contacts/__openerp__.py]: Odoo ?

File: addons/contacts/__openerp__.py [hunk 23]
```diff
@@ -0,0 +1,44 @@
...
+    'summary': 'Contacts, People and Companies',
+    'depends': [
+        'mail',
+    ],
+    'data': [
+        'contacts_view.xml',
+    ],
+    'installable': True,
+    'application': True,
+    'auto_install': False,
```
Comment (rim-odoo) [inline addons/contacts/__openerp__.py]: auto_install: False is the default value

File: addons/web/views/webclient_templates.xml (hunk 18 above)
Comment (tivisse) [inline addons/web/views/webclient_templates.xml]: Will be done in the new design task.

File: addons/contacts/__openerp__.py [hunk 24]
```diff
@@ -19,4 +19,23 @@
...
-import models
+{
+    'name': 'Address Book',
+    'version': '1.0',
+    'category': 'Tools',
+    'description': """
+This module gives you a quick view of your address book, accessible from your home page.
+You can track your suppliers, customers and other contacts.
+""",
+    'author': 'OpenERP SA',
```
Comment (rim-odoo) [inline addons/contacts/__openerp__.py]: Odoo SA ? ;-)

**Pull request closed without merging**